src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,165 @@
+<stache
+  pageTitle="Back button"
+  navTitle="Back button"
+  showTableOfContents="true">
+
+  <stache-page-anchor>
+    New page behavior
+  </stache-page-anchor>
+
+  <p>
+    Actions where a user navigates to a new context, such as a new record, list, or section, should create new entries in the browser history with a new page title. The new entry enables users to select the browser's back button and return to the previous page or view.
+  </p>
+  <p>
+    As a rule of thumb, create a new page in the browser history if the majority of the page content changes or a user would perceive a signficant change of context. Don't create a new URL or new browser entry for inline interactions such as steps in a wizard or opening a detail panel.
+  </p>
+
+  <div class="sky-subsection-heading">Focus</div>
+
+  <p>
+    After loading a new page, users should be able to use the tab key to reach the first focusable element on the page.
+  </p>
+
+  <div class="sky-subsection-heading">New page annoucement for screenreaders</div>
+
+  <p>
+    When a new page loads a SKY UX SPA, the page should behave similarly to what users would expect from a regular server-side application. The page should announce the page's &lt;title&gt; to the screen reader and inform users the page loaded via an ARIA live region when the &lt;title&gt;element is ready.
+  </p>
+  <p>
+    Behaviors vary based on different browser and screen reader combinations, but SKY UX applications use the combination listed above.
+  </p>
+
+  <stache-page-anchor>
+    Tabs and vertical tabs as site level navigation
+  </stache-page-anchor>
+
+  <p>
+    If you use tabs as page navigation, each tab should be a page in the browser history, and users should be able to use the back button to return to the previous view. Tabs contain all of a page's content and can't be dismissed.
+  </p>
+
+  <div class="screenshot screenshot-info">
+    <img src="~/assets/img/guidelines/back-button/tabs-backbutton.png" alt="Tabs as page navigation" style="width: 100%" />
+  </div>
+
+  <div class="sky-subsection-heading" style="margin-top: 20px;">Focus</div>
+
+  <p>
+    Based on guidance from <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel">W3C's WAI-ARIA Authoring Practices for tabs</a>, maintain focus on activated tabs.
+  </p>
+
+  <div class="sky-subsection-heading">New page announcement for screenreaders</div>
+
+  <p>
+    Titles are announced as described in <a stacheRouterLink="." fragment="new-page-behavior">New page annoucement for screenreaders</a>.
+  </p>
+
+  <stache-page-anchor>
+    Tabs as containers for viewing items
+  </stache-page-anchor>
+
+  <p>
+    If tabs contain removable items such as lists or reports, don't include each tab in the browser history.
```

Comment:
Same question on whether "tabs" at the start should be "tabs and vertical tabs." 

Also, I'd change the last part of the sentence to "... don't create entries in the browser history" to use the same terminology as we used above.